source/MaterialXView/Viewer.cpp

```diff
@@ -24,6 +24,8 @@ const int DEFAULT_ENV_SAMPLES = 16;
 
 namespace {
 
+mx::StringVec skippedXIncludes;
```

Comment:
Instead of keeping this in an anonymous namespace, it's better to keep this as a data member on the Viewer. 

Also can we change the name to something like "includeFiles" or "xincludeFiles" ? Thanks.